src/pages/learning/go.jsx

```diff
@@ -21,6 +27,35 @@ const LearningLandingPage = () => {
   };
 
   const curriculumSize = GitcoinCurriculumData.length;
+  const web3 = createAlchemyWeb3('https://eth-rinkeby.alchemyapi.io/v2/XW3eK_0nzE7TCKgZ589OxC94gNQrYJyW'); 
```

Comment:
@raykanani this URL is for our ethereum node through alchemy. I keep trying to hide it through our env file but to no avail. Could I get your help on this?